docs/sdn_controller.md

```diff
@@ -1,17 +1,15 @@
 # SDN Controller
 
-> SDN Controller is available in XOA 5.36.0 and higher
+The SDN Controller enables a user to **create pool-wide and cross-pool private networks** and more. It's available as a Xen Orchestra plugin (embed in XOA Premium).
 
-The SDN Controller enables a user to **create pool-wide and cross-pool** (since XOA 5.38.0) **private networks**.
+## Global Private Networks
 
-![](./assets/sdn-controller.png)
+![](https://xen-orchestra.com/blog/content/images/2019/08/gpn.png)
```

Comment:
IMHO, it's a bad idea to depend on external resources, especially for trivial things like pictures that could be duplicated here.

Reply:
Understood